apps/src/applab/designMode.js

```diff
@@ -188,7 +203,7 @@ designMode.onPropertyChange = function(element, name, value) {
     default:
       throw "unknown property name " + name;
   }
-  Applab.levelHtml = designMode.serializeToLevelHtml();
+  designMode.serializeToLevelHtml();
```

Comment:
agreed, there is no point in serializing if we are not also saving (i.e. on toggle, run, or autosave)